src/frontend/Komponenter/Behandling/Brevmottakere/baks/BrevmottakerContainer.tsx

```diff
@@ -44,12 +45,14 @@ export function BrevmottakerContainer({ behandlingId }: Props) {
         });
     }
 
-    async function opprettBrevmottaker(nyBrevmottaker: NyBrevmottaker): Promise<boolean> {
-        return await axiosRequest<Brevmottaker[], NyBrevmottaker>({
+    async function opprettBrevmottaker(
```

Comment:
Ikke sånn superviktig men det kan være nice å skrive seg bort fra å deklarere funksjoner med `function`-nøkkelordet og heller bruke `arrow functions`.

Dette er fordi `arrow functions` er en mer moderne metode å lage funksjoner i js men mest av alt for å unngå å deklarere metoder på to forskjellige måter i appen :)